pkgs/applications/audio/easyeffects/default.nix

```diff
@@ -4,11 +4,12 @@
 , fetchFromGitHub
 , calf
 , fftwFloat
+, fmt
 , glib
 , glibmm
```

Comment:
This one too.

Reply:
> anonymous function at /home/jtojnar/Projects/nixpkgs/pkgs/applications/audio/easyeffects/default.nix:1:1 called with unexpected argument 'glibmm'

means that you also need to remove `glibmm` from being passed in `all-packages.nix`.